spec/models/canonical/armor_spec.rb

```diff
@@ -189,29 +189,44 @@
       end
     end
 
-    describe 'smithing materials' do
+    describe 'crafting materials' do
       let(:armor) { create(:canonical_armor) }
-      let(:material) { create(:canonical_raw_material) }
 
       before do
-        armor.canonical_craftables_crafting_materials.create!(material:, quantity: 4)
+        armor.canonical_crafting_materials.create!(
+          quantity: 4,
+          source_material: create(:canonical_raw_material),
+        )
+        armor.reload
       end
 
       it 'gives the quantity needed' do
```

Comment:
We need to update this spec since it no longer tests anything meaningful. It should test that the crafting materials for the in-game armour item should be the same as for the corresponding canonical model.

Reply:
My b, this IS the canonical model. So these tests just really aren't needed anymore.